src/test/java/io/opentelemetry/OverheadTests.java

```diff
@@ -56,25 +54,48 @@ static void tearDown() {
     collector.close();
   }
 
-  @Disabled
-  @TestFactory
-  Stream<DynamicTest> runAllTestConfigurations() {
-    return Configs.all().map(config ->
-        dynamicTest(config.getName(), () -> runTestConfig(config))
-    );
+  @Test
```

Comment:
We should really delete this version probably and only have the externals version. I suspect this hasn't been run for a long time anyway....even by humans.

```suggestion
  @Test
  @Disabled
```